docs/getting_started/basic_tests.rst

```diff
@@ -33,6 +33,37 @@ gives access to the following parameters:
 - ``case_id``: current test case identification number (optional for test suite setup and teardown)
 - ``aux_list``: list of used auxiliaries
 
+
+``suite_id`` and ``case_id`` are used to coordinate and define a clear test execution order.
+It is now optional for ``pykiso.BasicTest`` but still mandatory in case of ``pykiso.RemoteTest``.
```

Comment:
```suggestion
It is now optional for ``pykiso.BasicTest`` but still mandatory in case of ``pykiso.RemoteTest`` as the TestApp protocol relies on these identifiers.
```
(maybe with a ref to the testapp protocol)